index.html

```diff
@@ -0,0 +1,110 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Mint's Sign Up Page</title>
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.6.3/css/all.css" integrity="sha384-UHRtZLI+pbxtHCWp1t77Bi1L4ZtiqrqD80Kn4Z8NTSRyMA2Fd33n5dQ8lWUE00s/" crossorigin="anonymous">
+</head>
+<body>
+    <div id="main-wrapper">
+        <header>
+            <div id="signin-link">
+                <div id="forgot-pwd">
+                    <span>Already have an Intuit account?</span>
+                    <span><a href="#">I forgot my User ID or Password</a></span>
+                </div>
+                <div id="signin-btn">
+                    <a href="#">
+                        <button id="signin">
```

Comment:
The element button must not appear as a descendant of the `<a>` element.

Reply:
removed <a> element completely